webdriver-spec.html

```diff
@@ -699,13 +699,18 @@
 <section>
 <h2>Interface</h2>
 
-<pre class=idl>partial interface Navigator {
-	[Unforgeable] readonly attribute boolean webdriver;
+<p>If the <dfn>webdriver-active flag</dfn> is true,
```

Comment:
Is it possible for this flag to change at runtime, i.e. is it possible to use WebDriver to connect to an already-running browser?